docs/stencil-docs/storefront-customization/ADA compliance-levels.md

```diff
@@ -0,0 +1,301 @@
+
+Guideline 3.2.2 On Input: Changing the setting of any user interface component does not automatically cause a change of context unless the user has been advised of the behavior before using the component.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone ensures that you must click a submit button to perform the action.
+ 
+</div>
+</div>
+</div>
+
+**Understandable > Consistent Navigation: Level AA**
+
+Guideline 3.2.3 Consistent Navigation: Navigational mechanisms that are repeated on multiple web pages within a set of web pages occur in the same relative order each time they are repeated, unless a change is initiated by the user.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+* Cornerstone uses a consistent presentation and layout for users who interact with repeated content.
+* Cornerstone navigation menus are always in the same place.
+* The Search box is in the same location on all pages.
+ 
+</div>
+
+**Understandable > Change on Request: Level AAA**
+
+Guideline 3.2.5 Change on Request: Changes of context are initiated only by user request or a mechanism is available to turn off such changes.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone automatically redirects a user from an old page to a new page in such a way that he or she never realizes the redirect has occurred.
+ 
+</div>
+</div>
+</div>
+
+**Understandable > Error Identification: Level A**
+
+Guideline 3.3.1 Error Identification: If an input error is automatically detected, the item that is in error is identified and the error is described to the user in text.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone indicates if the user fails to include an `@` in the email address.
+ 
+</div>
+</div>
+
+
+**Understandable > Error Prevention: Level AA**
+
+Guideline 3.3.4 Error Identification: For web pages that cause legal commitments or financial transactions for the user to occur, that modify or delete user-controllable data in data storage systems, or that submit user test responses, at least one of the following is true: Reversible, Checked, and Confirmed.
+
+**Cornerstone Technique:**
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+Cornerstone provides a checkbox in addition to a submit button.
+ 
+</div>
+</div>
+</div>
+
+**Understandable > Help: Level AAA**
+
+Guideline 3.3.5 Help: For web pages that cause legal commitments or financial transactions for the user to occur, that modify or delete user-controllable data in data storage systems, or that submit user test responses, at least one of the following is true: Reversible, Checked, and Confirmed.
```

Comment:
The text is a duplicate of Guideline 3.3.4 Error Identification. Is that correct?

Reply:
Thank you for catching this one.